test/extended/operators/tolerations.go

```diff
@@ -0,0 +1,77 @@
+package operators
+
+import (
+	"fmt"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = Describe("[Feature:Platform][Smoke] Managed cluster", func() {
+	oc := exutil.NewCLIWithoutNamespace("operators")
+
+	It("should ensure pods do not tolerate all taints unless on whitelist", func() {
+		// iterate over the references to find valid images
+		pods, err := oc.KubeFramework().ClientSet.CoreV1().Pods("").List(metav1.ListOptions{})
+		if err != nil {
+			e2e.Failf("unable to list pods: %v", err)
+		}
+
+		// list of pods that use images not in the release payload
+		invalidPodTolerations := sets.NewString()
```

Comment:
How about collecting a list of all pod tolerations and printing that out, so that we have an accounting for everything used (which might possibly include typos) for at least initial review?